### ISSUE TITLE
Paypal Error

### ISSUE THREAD

**Issue by ekobimantara** [woocommerce/woocommerce-paypal-payments#1525]
Hi, Im using Paypal Debit and Credit Card payment's woocommerce plugin plugin in my website; https://ekobimantara.com/, and currently it can't be used, it said "Something went wrong. Please try another payment source" when click the paypal or credit/debit card button.

**Comment by ekobimantara** [comment]
Hi, Im using Paypal Debit and Credit Card payment's woocommerce plugin plugin in my website; https://ekobimantara.com/, and currently it can't be used, it said "Something went wrong. Please try another payment source" when click the paypal or credit/debit card button.

**Comment by Sirvijver** [comment]
Hello @ekobimantara 

You have reached out to open GitHub forum with a support request. We utilize GitHub mainly for development activities, so I would like to ask you to [reach out to the support team](https://woocommerce.com/document/woocommerce-paypal-payments/#get-help) directly for more guidance.

I have reviewed your checkout page and see that you are using 2 PayPal Plugins. This very often causes issues, and we advise everybody to use our plugin, to not have another PayPal plugin active on their site. These plugins utilize similar resources and easily cause failures.

However, when I tested your checkout page, it did not show the error you reported. If you do still have such an issue, I would advise you to reach out to our support team via the link provided above. This thread will now be closed.

Kind regards, 
Joost
